connect_ext_ppr/webapp.py

```diff
@@ -465,6 +482,74 @@ def list_hubs_by_product(
             reponse_list.append(HubSchema(id=hub['id'], name=hub['name']))
         return reponse_list
 
+    @router.get(
+        '/deployments/{deployment_id}/pricing/batches',
+        summary='List Pricing Batch for Deployment',
+        response_model=List[BatchSchema],
+    )
+    def get_deployment_batches(
+        self,
+        deployment_id: str,
+        db: VerboseBaseSession = Depends(get_db),
+        client: ConnectClient = Depends(get_installation_client),
+        installation: dict = Depends(get_installation),
+    ):
+        deployment = get_deployment_by_id(deployment_id, db, installation)
+        marketplace_ids = identify_marketplaces(
+            client,
+            deployment.hub_id,
+        )
+
+        batches = list(client('pricing').batches.filter(
+            R().stream.owner.id.eq(deployment.account_id),
+            R().stream.context.product.id.eq(deployment.product_id),
+            R().stream.context.marketplace.id.in_(marketplace_ids),
+            R().test.ne(True),
+            R().status.eq('published'),
+        ))
```

Comment:
[SOL-C] I would add an order here by `events.created.at`

Reply:
I think its not needed as batches are by default shorted by creation date.
<img width="516" alt="image" src="https://github.com/cloudblue/connect-extension-xvs/assets/1707894/9b8ca6d0-378d-45b8-bdc4-37243365df3f">